README.md

```diff
@@ -1,9 +1,9 @@
 ## Google Cloud Operations integration for GKE
 
-> :exclamation: **Tools in this repository are not meant for 
-> end-users.**
-> It contains source code for tools that are pre-installed in 
-> the GKE clusters.
+> :exclamation: **Most tools in this repository are not meant for end-users.**
+> It contains source code for tools that are pre-installed in the GKE clusters.
+> An exception to this is the [Custom Metrics - Stackdriver Adapter](custom-metrics-stackdriver-adapter),
+> which is meant for end-user consumption.
```

Comment:
Perhaps we should refer back to https://cloud.google.com/kubernetes-engine/docs/tutorials/autoscaling-metrics? That contains the instructions needed to actually make use of the cmsa.

(Mostly, I think we expect people to go from that page to here, rather than the other way around. But for anyone landing here first, having that context would be useful.)

lgtm either way though.

Reply:
Good idea. Done.